style.css

```diff
@@ -52,3 +52,20 @@ a {
 .is-style-arrow-icon-details[open] > summary {
 	list-style-type: '\2192\00a0\00a0\00a0';
 }
+
+/*
+ * Styles for the taxonomy/tags on single posts
+ * https://github.com/WordPress/twentytwentyfour/issues/XX
+ */
+
+.wp-block-post-terms.taxonomy-post_tag a {
```

Comment:
Let's make this a block style so users can turn it off

Reply:
This is now using a block style 👍